tests/test_neurips_conference_v2.py

```diff
@@ -1178,1648 +1173,1017 @@ def test_post_submission_stage(self, helpers, openreview_client, client, request
-#             replyto=submissions[0].id,
-#             readers=['NeurIPS.cc/2023/Conference/Program_Chairs', 'NeurIPS.cc/2023/Conference/Paper5/Senior_Area_Chairs', 'NeurIPS.cc/2023/Conference/Paper5/Area_Chairs'],
-#             writers=['NeurIPS.cc/2023/Conference/Paper5/Senior_Area_Chairs'],
-#             signatures=['NeurIPS.cc/2023/Conference/Paper5/Senior_Area_Chairs'],
-#             content={
-#                 'title': 'Test an SAC comment',
-#                 'comment': 'This is an SAC comment'
-#             }
-#         ))
+        submissions = openreview_client.get_notes(invitation='NeurIPS.cc/2023/Conference/-/Submission', sort='number:asc')
+        reviews = pc_client_v2.get_notes(invitation='NeurIPS.cc/2023/Conference/Submission1/-/Official_Review')
+        review = reviews[0]
 
-#         helpers.await_queue()
+        assert len(openreview_client.get_invitations(invitation='NeurIPS.cc/2023/Conference/-/Rebuttal')) == 2
+        invitation = openreview_client.get_invitation(f'{review.signatures[0]}/-/Rebuttal')
+        assert invitation.maxReplies == 1
+        assert invitation.edit['note']['replyto'] == review.id
 
-#         process_logs = client.get_process_logs(id=comment_note.id)
-#         assert len(process_logs) == 1
-#         assert process_logs[0]['status'] == 'ok'
+        test_client = openreview.api.OpenReviewClient(username='test@mail.com', password=helpers.strong_password)
 
-#         messages = client.get_messages(to='sac1@google.com', subject='[NeurIPS 2023] Your comment was received on Paper Number: 5, Paper Title: \"Paper title 5\"')
-#         assert messages and len(messages) == 1
+        rebuttal_edit = test_client.post_note_edit(
+            invitation=f'{review.signatures[0]}/-/Rebuttal',
+            signatures=['NeurIPS.cc/2023/Conference/Submission1/Authors'],
+            note=openreview.api.Note(
+                replyto = review.id,
+                content={
+                    'rebuttal': { 'value': 'This is a rebuttal reply to an official review.' }
+                }
+            )
+        )
 
-#     def test_rebuttal_stage(self, conference, helpers, test_client, client):
+        with pytest.raises(openreview.OpenReviewException, match=r'.*You have reached the maximum number \(1\) of replies for this Invitation.*'):
+            rebuttal_edit = test_client.post_note_edit(
+                invitation=f'{review.signatures[0]}/-/Rebuttal',
+                signatures=['NeurIPS.cc/2023/Conference/Submission1/Authors'],
+                note=openreview.api.Note(
+                    replyto = review.id,
+                    content={
+                        'rebuttal': { 'value': 'This is another rebuttal reply to the same official review.' }
+                    }
+                )
+            )
 
-#         now = datetime.datetime.utcnow()
-#         due_date = now + datetime.timedelta(days=3)
+        # use custom stage to enable one author rebuttal per paper
```

Comment:
@melisabok I think it's a better idea to use the custom stage to enable one rebuttal per paper. If we re-use the rebuttal stage, then all rebuttal invitations will have '/-/Rebuttal' as the super invitation and that might cause confusion.

Reply:
ohhh I see, ok, let's use the custom stage then.

Reply:
there is no way to customize the name, right? because if you use another name then there will be two different super invitation. Anyway is fine with me.

Reply:
There isn't a way to customize it from the request form, but the stage does have a `name` param, so perhaps I should just add it to the request form. This way they can change the name and use the same button.